code/script.js

```diff
@@ -33,19 +46,124 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
+// Question flow
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userAnswer(message)
+    input.value = ''
+    setTimeout(() => dayBooking(message), 700)
+  } else if (questionNumber === 2) {
+    userAnswer(message)
+    setTimeout(() => timeBooking(message), 700)
+  } else if (questionNumber === 3) {
+    userAnswer(message)
+    setTimeout(() => guestBooking(message), 700)
+  } else if (questionNumber === 4) {
+    userAnswer(message)
+    setTimeout(() => finaliseBooking(message), 700)
+  } 
+}
+
+// First question
 const greeting = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
+  botAnswer(`Welcome! In what name do you want to make a reservation?`);
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000);
+form.addEventListener('submit', (event) => {
+  event.preventDefault()
+  const name = nameInput.value
+  userAnswer(`${name}`)
+  nameInput.value = ''
+  setTimeout(() => dayBooking(), 1000)
+})
+
+// second question dayBooking
+
+const dayBooking = () => { 
+  questionNumber++
+  botAnswer(`Which day would you like to visit us?`);
+
+  inputWrapper.innerHTML = `
+    <button id='fridayButton'>Friday</button>
+    <button id='saturdayButton'>Saturday</button>
+    `
+  document
+    .getElementById('fridayButton')
+    .addEventListener('click', () => nextQuestion('Friday'))
+  document
+    .getElementById('saturdayButton')
+    .addEventListener('click', () => nextQuestion('Saturday'))
+  }
+
+// third question
+
+const timeBooking = (type) => {
+  questionNumber++
+  botAnswer(`Which time on ${type} would you like to book?`)
+  if (type === 'Friday') {
+    inputWrapper.innerHTML=`
+    <select id = 'select'>
+      <option value='' selected disabled> Choose time </option>
+      <option value='Lunch'>Lunch</option>
+      <option value='After Work'>After Work</option>
+    </select> 
+    `
+  } 
+  else {
+    inputWrapper.innerHTML=`
+    <select id = 'select'>
+      <option value='' selected disabled> Choose time </option>
```

Comment:
Try to remove selected before disabled. We think it should only be disabled on this option. Code seems to work, but it might cause an error.